app/parsers.py

```diff
@@ -120,23 +120,32 @@ def parse_direct_challenge_queued(blob):
 @util.debug_log_trace
 def parse_sideboard_submit(blob):
     import app.mtga_app as mtga_app
+    app.mtga_app.mtga_logger.info("{}".format(pprint.pformat(blob)))
```

Comment:
(low pri / fyi, happy to merge without this change)
Missing `util.ld()` (tab depth by call stack) ....but using pformat here makes it weird. maybe something like:

```python
for line in pprint.pformat(blob).split("\n"):
    app.mtga_app.mtga_logger.info("{}{}".format(util.ld(), pprint.pformat(blob)))
```